_assets/js/templates/ellipsisTemplate.js

```diff
@@ -0,0 +1,8 @@
+const ellipsis = () => {
```

Comment:
Not sureeeee we can use ES6.  Until we get that build process built, we might want to use the old school for legacy browsers.